test/pdfs/issue19182.pdf.link

```diff
@@ -0,0 +1 @@
+https://github.com/user-attachments/files/18037940/ExtnB.character.not.working.pdf
```

Comment:
Given that the PDF is very short and simple, could we perhaps just add it directly to the repository instead?